internal/grpctest/grpctest.go

```diff
@@ -98,8 +98,13 @@ func RunSubTests(t *testing.T, x interface{}) {
 		tfunc := getTestFunc(t, xv, methodName)
 		t.Run(strings.TrimPrefix(methodName, "Test"), func(t *testing.T) {
 			setup(t)
-			// defer teardown to guarantee it is run even if tfunc uses t.Fatal()
-			defer teardown(t)
+			// Run leakcheck in t.Cleanup() to guarantee it is run even if tfunc
+			// uses t.Fatal().
```

Comment:
Should this go above `setup` for the same reason?

Reply:
Moved.